### PR TITLE
Yoav - Fix talons code

### PR DESCRIPTION
**Pull request by hillelv**

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/elbow/falconElbow/FalconElbowConstants.java [hunk 1]
```diff
@@ -2,17 +2,21 @@
...
 
-import static edu.greenblitz.robotName.subsystems.arm.elbow.ElbowConstants.BACKWARD_ANGLE_LIMIT;
-import static edu.greenblitz.robotName.subsystems.arm.elbow.ElbowConstants.FORWARD_ANGLE_LIMIT;
+import static edu.greenblitz.robotName.subsystems.arm.elbow.ElbowConstants.*;
 
 public class FalconElbowConstants {
 
     public static final int MOTOR_ID = 23;
 
+    public static final String CANBUS_NAME = "*";
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/elbow/falconElbow/FalconElbowConstants.java:16]:
```suggestion
    public static final String CANIVORE_NAME = "*";
```

File: src/main/java/edu/greenblitz/robotName/subsystems/shooter/pivot/falconPivot/FalconPivotConstants.java [hunk 2]
```diff
@@ -8,6 +8,8 @@
 public class FalconPivotConstants {
 	
 	public static final int MOTOR_ID = 12;
+
+	public static final String CANBUS_NAME = "*";
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/shooter/pivot/falconPivot/FalconPivotConstants.java:12]: same here about the name

File: src/main/java/edu/greenblitz/robotName/utils/sysId/falconSysId/SysIdFalconConstants.java [hunk 3]
```diff
@@ -4,14 +4,15 @@
...
+    protected static final double SIGNAL_SPEED = 250;
 
-    public static final double DYNAMIC_VOLTAGE = 4;
+    protected static final double DYNAMIC_VOLTAGE = 4;
 
-    public static final int MOTOR_ID = 187;
+    protected static final int MOTOR_ID = 187;
 
-    public static final SoftwareLimitSwitchConfigs LIMIT_SWITCH_CONFIGS = new SoftwareLimitSwitchConfigs();
+    protected static final String CANBUS_NAME = "*";
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/utils/sysId/falconSysId/SysIdFalconConstants.java:13]: same here about the name

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/greenblitz/robotName/utils/sysId/falconSysId/SysIdFalconConstants.java (hunk 3 above)
Comment (bossyshrimp68) [reply src/main/java/edu/greenblitz/robotName/utils/sysId/falconSysId/SysIdFalconConstants.java:13]: no , i wiil explain about it when you will finis your conversation with shachar, noam, yoda, broder and raz

**Review by bossyshrimp68 — APPROVED** [review]

**Review by aaa-png — APPROVED** [review]

File: et HEAD@{1}
Comment (tnrz135) [inline et HEAD@{1}:1]: Why is this here

**Review by tnrz135 — CHANGES_REQUESTED** [review]

**Review by tnrz135 — APPROVED** [review]
